File: Backend/Iteramico/DomainRepository/Models/City.cs
namespace DomainRepository.Models
{
#nullable disable
    public class City
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string LocalName { get; set; }

        public int CountryId { get; set; }
        public virtual Country Country { get; set; }
    }
}

File: frontend/lib/models/city.dart
import 'package:frontend/models/country.dart';

class City {
  final int id;
  final String localName;
  final String name;
  final Country country;

  City(
      {required this.id,
      required this.localName,
      required this.name,
      required this.country});

  factory City.fromJson(Map<String, dynamic> json) {
    return City(
      id: json['id'],
      name: json['name'],
      localName: json['localName'],
      country: Country.fromJson(json['country']),
    );
  }
}
